compiler/packages/react-mcp-server/src/index.ts

```diff
@@ -356,7 +356,7 @@ Server Components - Shift data-heavy logic to the server whenever possible. Brea
 
 server.tool(
   'review-react-runtime',
-  'Review the runtime of the code and get performance data to evaluate the proposed solution, the react code that is passed into this tool MUST contain an App component.',
+  'Review the runtime of the code and get performance data to evaluate the proposed solution, the react code that is passed into this tool MUST contain an App functional component without arrow function. And DO NOT export anything',
```

Comment:
Can you also add something like "run this tool every time you propose a performance related change to verify if your suggestion actually improves performance"